### PR TITLE
Glossary shrinked to two columns

### LINKED ISSUE #2041: Glossary field scrolls outside of box when zooming webpage
110%
skjermbilde pa 2018-05-24 22-20-15

120%
skjermbilde pa 2018-05-24 22-20-37

130%
skjermbilde pa 2018-05-24 22-20-53

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/58797511-glossary-field-scrolls-outside-of-box-when-zooming-webpage?utm_campaign=plugin&utm_content=tracker%2F253393&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F253393&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Naruto0**
Fixes #2041

Signed-off-by: Filip Hron <filip.hron@gmail.com>

### CODE REVIEW

File: weblate/templates/glossary-embed.html [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
+    </td>
+    </tr>
+    </tr>
+{% empty %}
+    <tr>
+    <td colspan="2">
+{% trans "No related strings found in the glossary." %}
+    </td>
+    </tr>
+{% endfor %}
```
[the remark this answers is not in the document]
Comment (Naruto0) [reply weblate/templates/glossary-embed.html]: I may changed the end of lines, yes.  I will fix it.

**Comment by nijel** [comment]
There are several problems with this, but I'll address them separately as when investigating this I anyway had to figure out solution.

- there is wrong HTML markup leading to wrong rendering in some cases (extra table cells in footer)
- the copy button no longer works due to changed markup

**Comment by Naruto0** [comment]
> There are several problems with this, but I'll address them separately as when investigating this I anyway had to figure out solution.
> 
> * there is wrong HTML markup leading to wrong rendering in some cases (extra table cells in footer)
> * the copy button no longer works due to changed markup

Hi @nijel while fiddeling i too did observethat the change disabled the button functionality. Haven't figured out something better yet.

Regards

**Comment by nijel** [comment]
See my fixes:

* f4c3135ff9 Remove copy button from glossary
* 006bbbc792 Avoid having doble form and csrf in translate page
* 6de49d2e65 Fix markup after #2266

**Pull request merged**